tracer/src/Datadog.Trace/Configuration/Schema/MessagingSchema.cs

```diff
@@ -89,5 +89,11 @@ public MsmqTags CreateMsmqTags(string spanKind)
                 SchemaVersion.V0 when !_peerServiceTagsEnabled => new MsmqTags(spanKind),
                 _ => new MsmqV1Tags(spanKind),
             };
+
+        public AwsSqsTags CreateAwsSqsTags(string spanKind) => _version switch
```

Comment:
There was a good point raised internally by Zarir: Do we want to create the peer.service tags for inbound spans (i.e. `spanKind == "consumer"`)? cc @pablomartinezbernardo 

If not, then we'll need to do one of the following:
- Return the older AwsSqsTags in this scenario: Add another line to the switch expression like `_ when spanKind == SpanKinds.Consumer => new AwsSqsTags(spanKind)`
- Update AwsSqsV1Tags to return `null` for `PeerService`/`PeerServiceSource` when SpanKind is "consumer"

Reply:
For the record, if we need to change this, I support doing this in a separate PR. No need to block all the changes on what may be a small followup PR

Reply:
Will do in another PR. 

FYI, we currently do create `peer.service` in SQS triggers in an AWS Lambda's inferred spans, which could be considered as "consumers" too.

Reply:
I think the safest bet is definitely ignoring inbound flows. Not sure at all of the impact including peer.service in inbound spans will have, but we do know that other tracers have not been including it.

Can you give more detail on what you mean by this?

> FYI, we currently do create peer.service in SQS triggers in an AWS Lambda's inferred spans, which could be considered as "consumers" too.

I already removed it from my rabbit PR, and I will take care of removing it from the Kafka implementation as well.

Reply:
Inferred spans are inbound spans that get generated on the Datadog Lambda Extension (flavor of the agent for Lambda) which uses trace propagation between Lambda workloads. 

I just wanted to share it since it's relevant to `peer.service`, but it's non-actionable in the tracers side.